web/profiles/custom/os2loop/themes/os2loop_theme/package.json

```diff
@@ -21,6 +21,8 @@
   },
   "dependencies": {
     "bootstrap": "^4.6.0",
-    "bootstrap-icons": "^1.4.0"
+    "bootstrap-icons": "^1.4.0",
+    "jquery": "^3.6.0",
+    "popper.js": "^1.16.1"
```

Comment:
We should add these as `devDependencies` since everything we load is built with Webpack (and we don't have the `node_modules` folder at runtime).